ceremonies/rpc/src/lib.rs

```diff
@@ -66,13 +134,25 @@ where
 	) -> Result<Vec<(CeremonyIndexType, CommunityReputation)>> {
 		self.deny_unsafe.check_if_safe()?;
 
-		let api = self.client.runtime_api();
-		let at = BlockId::hash(at.unwrap_or_else(|| self.client.info().best_hash));
-		return Ok(api.get_reputations(&at, &account).map_err(runtime_error_into_rpc_err)?)
+		if !self.offchain_indexing {
+			return Err(offchain_indexing_disabled_error("ceremonies_getReputations"))
+		}
+
+		if self.cache_dirty(&reputation_cache_dirty_key(account.encode())) {
+			self.refresh_reputation_cache(account.clone(), at);
+		}
+
+		let cache_key = &reputation_cache_key(account.encode());
```

Comment:
No need to use `.encode()` here, we don't need to double encode, just pass the account as is. Please remove that everywhere.

Reply:
I could imagine that you can even pass a `&AccountId` to the `reputation_cache_key` function, might be worth a try.

Reply:
yep works